lib/chef/resource/support/client.erb

```diff
@@ -22,6 +21,7 @@
 <% next if instance_variable_get(prop).nil? || instance_variable_get(prop).empty? -%>
 <%=prop.delete_prefix("@") %> <%= instance_variable_get(prop).inspect %>
 <% end -%>
+minimal_ohai <%= @minimal_ohai.inspect %>
```

Comment:
So this means we have to do the same for all params with bool values, right?

Reply:
In theory, `boolean_value.inspect` should be fine, because it's effectively a `.to_s`, but I tend to view `.inspect` as not having a stable output (i.e., `<TrueClass: true>` or the like would be a valid output that future Ruby might decide on). I'd recommend `.to_s` instead.